CONTRIBUTING.md

```diff
@@ -204,18 +203,25 @@ any public functionality that is dependent on an optional dependency should be i
 `import_optional` function. 
 
 #### Note:
-- The `import_optional` function mirrors the python import functionality and thus its return type has to be `Any`. Using 
-objects imported with this function can lead to misspecification of types as `Any` when the developer intended to be 
-more restrictive. If you want to type a variable using a class that depends on an optional dependency then you should 
-use the `TYPE_CHECKING` to import it instead. For instance:
+- The `import_optional` function returns an object instance rather than an object. This will cause typechecking to fail 
```

Comment:
I find this kind of confusing as everything is an object and "object instance" is more confusing still. Should we replace `object instance` with `instance` and `object` with `class`?